rust_src/src/frames.rs

```diff
@@ -34,9 +41,12 @@ impl LispFrameRef {
         !self.terminal.is_null()
     }
 
-    // Awaiting Wilfred#1264
+    // Dragged from https://github.com/remacs/remacs/pull/1264
```

Comment:
This comment can simply be dropped. Once #1264 is merged this comment will have no meaning.